Marsiple/App/New Group/AlbumsViewController.swift

```diff
@@ -29,25 +35,16 @@ extension AlbumsViewController: UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return albums.count
+        return photoCollectionViewControllers.count
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: "UITableViewCell", for: indexPath) as UITableViewCell
-        return cell
-    }
-}
-
-extension AlbumsViewController: UITableViewDelegate {
-    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
         cell.addSubview(photoCollectionViewControllers[indexPath.row].view)
```

Comment:
We should probably add the `PhotoCollectionViewController` into the `cell`'s `contentView`.